tensorflow/tools/graph_transforms/quantize_nodes.cc

```diff
@@ -84,6 +84,13 @@ const std::vector<QuantizedOpInfo>& GetQuantizedOpList() {
        DT_QINT32,
        {},
        QuantizedOpInfo::CONTIGUOUS_MIN_MAX},
+      {"Conv2DBackpropInput",
+       {"strides", "padding"},
+       {{"Tinput", DT_QUINT8}, {"Tfilter", DT_QUINT8}, {"out_type", DT_QINT32}},
+       DT_QUINT8,
+       DT_QINT32,
+       {},
```

Comment:
Can you also add a simple test to https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/graph_transforms/quantize_nodes_test.cc#L315